build.gradle

```diff
@@ -32,8 +33,13 @@ buildscript {
 plugins {
   id 'com.gorylenko.gradle-git-properties' version '2.4.0-rc2'
   id 'com.github.johnrengelman.shadow' version '6.1.0'
+  id "com.palantir.docker" version "0.34.0"
+  // https://blog.ltgt.net/javax-jakarta-mess-and-gradle-solution/
+  // TODO id "org.gradlex.java-ecosystem-capabilities" version "1.0"
```

Comment:
I read the blog post partially. But not clear whether we need this or not?

Reply:
It is a reminder to revisit this. When GMS starts there is a massive block of warnings and delay like the following with like `2022-12-22 07:52:37 2022-12-22 13:52:37.499:WARN:oeja.AnnotationParser` this would address all those conflicts.